test/shared/src/main/scala/zio/test/Spec.scala

```diff
@@ -47,6 +47,19 @@ final case class Spec[-R, +E, +L, +T](caseValue: SpecCase[R, E, L, T, Spec[R, E,
       case c @ TestCase(_, _)         => f(c)
     }
 
+  /**
+   * Traverses all nodes and leaves only the ones that have a path to tests that satisfy the given predicate.
```

Comment:
Isn't this false? Don't we keep all the suites, regardless of whether they contain tests that satisfy the predicate?

Reply:
Yeah, sorry, the wording isn't accurate. Hmm... maybe "Traverses all nodes and filters out TestCases with labels that do not satisfy the given predicate."? Do you have better suggestions?

Reply:
"Returns a new Spec containing only tests with labels satisfying the specified predicate"?

Reply:
I like that better. Thanks. 👍